packages/valory/skills/mech_interact_abci/models.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.mech_contract_address: str = self._ensure(
             "mech_contract_address", kwargs, str
         )
+        self.request_price: Optional[int] = kwargs.get("request_price", None)
```

Comment:
Why not `self._ensure(...`?

Reply:
Maybe this is used in more than one skill and ensure pops.

Reply:
Can you point out which skill?

Reply:
That was a guess, not an actual assertion.

Reply:
It was decided that request_price be an optional parameter. Therefore, cannot use _ensure.

Reply:
This is an optional param.